src/ml_board/frontend/Dashboard/src/worker_socket/WorkerSocket.ts

```diff
@@ -0,0 +1,107 @@
+import socketIO, { Socket } from 'socket.io-client';
```

Comment:
this code is very well structured. Higher level functions call lower level functions and therefore it reads like a story. Good work! :-)

Reply:
Thanks, that's the whole reason why I have redone it :D glad you like it